src/main/java/com/microsoft/graph/authentication/InternalAuthConstants.java

```diff
@@ -0,0 +1,12 @@
+package com.microsoft.graph.authentication;
+
+/**
+ * Internal only.
```

Comment:
What does "Internal" mean here? If it means "not public", I think we should change the protection level to `protected` and drop `Internal` from the name.

Reply:
I have tried that before. The problem is that the compiler for android adds synthetic accessors in that case (and complains at linting) which impacts performance at runtime.

Reply:
I actually ended up moving those constants into the authentication provider. The only reason they were in a dedicated class I believe was because they were used for testing.